src/axom/core/memory_management.hpp

```diff
@@ -34,50 +41,40 @@ constexpr int INVALID_ALLOCATOR_ID = -1;
  * \param [in] resource_type the Umpire resource type
  * \return ID the id of the predefined umpire allocator.
  */
-inline int getResourceAllocatorID(
+inline int getUmpireResourceAllocatorID(
```

Comment:
Adding Umpire to the function name might confuse users that want to develop with and without umpire support since they would be calling ``getUmpireResouceAllocatorID``  even in code path that do not use umpire.

Perhaps revert to the previous name?

Reply:
This function is only available when `axom` is compiled with `umpire` and takes in an `umpire::resource::*`. In this case, I think adding the `Umpire` prefix is useful as it is more explicit about when it is appropriate to use this function.

Reply:
> This function is only available when axom is compiled with umpire

Thanks for clarifying @gzagaris -- that was hard to see from github.